workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/CustomerDashboard/pages/Apps/App/AppOutlet.tsx

```diff
@@ -115,11 +115,17 @@ const AppOutlet = () => {
 					OrderType.CLOUD
 				) {
 					const isDownloadableCloud =
-						product?.productSpecifications.some(
-							(specification) =>
+						product?.productSpecifications.some((specification) => {
+							if (
 								specification.specificationKey ===
-								PRODUCT_SPECIFICATION_KEY.DOWNLOADABLE_CLOUD_APP
-						);
+								PRODUCT_SPECIFICATION_KEY.APP_SETTINGS
+							) {
+								return (
+									JSON.parse(specification.value)
```

Comment:
Replace JSON.parse by this

```suggestion
									safeJSONParse(specification.value, {downloadableCloud: false})
```